sys/ztimer/periph_rtt.c

```diff
@@ -70,6 +70,8 @@ static const ztimer_ops_t _ztimer_periph_rtt_ops = {
     .set = _ztimer_periph_rtt_set,
     .now = _ztimer_periph_rtt_now,
     .cancel = _ztimer_periph_rtt_cancel,
+    .acquire = NULL,
```

Comment:
these lines are not needed, C implicitly initializes them to 0, which for all we know means NULL.